.vscode/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+      "target": "es5",
```

Comment:
I'm not sure where this came from, but es5 isn't our target, and please put .vscode/ in .gitignore